rdf_differ/entrypoints/ui/templates/index.html

```diff
@@ -10,19 +10,24 @@ <h1>List of calculated diffs</h1>
             <th scope="col">New version name</th>
             <th scope="col">Old version name</th>
             <th scope="col">Dataset URI</th>
+            <th scope="col">Actions</th>
         </tr>
         </thead>
         <tbody>
         {% for dataset in datasets %}
             {% if dataset %}
-            <tr>
-                <th scope="row">
-                    <a href="{{ url_for('view_dataset', dataset_id=dataset.dataset_id[1:]) }}">{{ dataset.dataset_id }}</a>
-                </th>
-                <td>{{ dataset.new_version_id }}</td>
-                <td>{{ dataset.old_version_id }}</td>
-                <td><a href="{{ dataset.dataset_uri }}" target="_blank">{{ dataset.dataset_uri }}</a></td>
-            </tr>
+                <tr>
+                    <th scope="row">
+                        <a href="{{ url_for('view_dataset', dataset_id=dataset.dataset_id[1:]) }}">{{ dataset.dataset_id }}</a>
```

Comment:
This looks odd - ```dataset.dataset_id[1:]```. Just curious, why everything else but the first one.

Reply:
it's a temporary measure used until we get to this [issue](https://meaningfy.atlassian.net/browse/MDR-169?atlOrigin=eyJpIjoiYjlmYzk5NjgxZWQwNDcxYWFhMDkzNGQxZWJhZTFhMjIiLCJwIjoiaiJ9), as the `dataset_id` value comes with a `/` in front, and I want to remove it before sending it to the view.

p.s.
it works with and without the `/` removed, but I wanted to maintain a concistency in our `urls`